lib/kitchen/directions/bake_notes/bake_numbered_notes.rb

```diff
@@ -45,18 +45,8 @@ def self.bake_note_exercise(note:)
         return unless solution
 
         # bake solution in place
-        exercise.add_class('os-hasSolution')
-        solution[:id] = "#{exercise[:id]}-solution"
-        solution_number = note.first('.os-number').text
-        solution.replace_children(with:
-          <<~HTML
-            <a class="os-number" href="##{exercise[:id]}">#{solution_number}</a>
-            <span class="os-divider"> </span>
-            <div class="os-solution-container">
-              #{solution.children}
-            </div>
-          HTML
-        )
+        Kitchen::Directions::BakeNumberedExercise.bake_solution_v1(
```

Comment:
this can probably lose the `Kitchen::Directions` prefix since we are in that scope already